webots_ros2_tutorials/worlds/custom_line_follower.wbt

```diff
@@ -0,0 +1,91 @@
+#VRML_SIM R2020b utf8
+WorldInfo {
+  info [
+    "Simulation for Video 6: Webots ros2 tutorial series on Soft-illusion youtube channel"
+    "Copyright (C) Soft-illusion"
+  ]
+  coordinateSystem "NUE"
+  title "ROS2 simulation of the custom robot for line following"
+}
+Viewpoint {
+  orientation -0.3058819481884525 -0.8679302660128931 -0.3913224848015066 1.9494067816584852
+  position -2.0268683243479075 1.933834895887204 -0.8320245040472168
+}
+TexturedBackground {
+}
+TexturedBackgroundLight {
+}
+RectangleArena {
+  floorSize 2 2
+  floorTileSize 2 2
+  floorAppearance PBRAppearance {
+    baseColorMap ImageTexture {
+      url [
+        "textures/loop_img.jpg"
```

Comment:
I would rename this file to `loop.jpg` as `img` is a useless abbreviation in this case.